com.snipaste.Snipaste.metainfo.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.snipaste.Snipaste</id>
+  <name>Snipaste</name>
+  <project_license>LicenseRef-proprietary=https://www.snipaste.com/index.html</project_license>
```

Comment:
https://www.snipaste.com/index.html doesn't go to the license.

Reply:
fixed